packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -542,7 +511,7 @@ export class AnchorSet implements ISubscribable<AnchorSetRootEvents> {
 	public applyDelta(delta: Delta.Root): void {
 		let parentField: FieldKey | undefined;
 		let parent: UpPath | undefined;
-		const moveTable = new Map<Delta.MoveId, UpPath>();
+		const moveTable = new Map<Delta.MoveId, { src: UpPath; nodes: PathNode[] }>();
```

Comment:
holding onto the `src: UpPath` here isn't a great pattern. This hold onto the UpPath across other edits, which means this path may or may not be valid when its reused (since some nodes it in might be the anchor nodes you are editing, and others might be constants which don't change: exactly which ones are is not clear from this type).

It looks like you only need the index offset for the start of this chunk within its parent field: it would be more clear to just store that, and would avoid making someone else modifying this code in the future think they could so something else with that path, which likely won't work due the issues I noted above.